README.md

```diff
@@ -3,8 +3,20 @@
 This project contains C# AltTester tests for a project using the New Input System.
 The tested actions are: tap object, key down/ key up, press key, click object, scroll, begin/end touch and tap simulated with swipe.
 
+## Before running the tests
+To run the tests, you must include the AltTester Unity SDK in the project. To do that, you can choose between the following ways:
+1. Add the AltTester Unity SDK submodule to the project
+    - in the <i> New Input System proj/Assets </i> folder add the submodule using the <i><b> git submodule add https://github.com/alttester/AltTester-Unity-SDK.git </b></i> command;
```

Comment:
For commands, you could use  ``  at the beginning and the end of the command too look like this: ``command``.